examples/features/sub-modules/inter-broker-bridge/README.md

```diff
@@ -0,0 +1,143 @@
+# Inter-broker Bridging
```

Comment:
Can we leave the original title, as it’s only a JMS bridge there are other ways to bridge between brokers that is not JMS and not just ActiveMQ 5.x